packages/block-library/src/playlist/index.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Server-side rendering of the `core/playlist` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/playlist` block on server.
+ *
+ * @since 6.8.0
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the Playlist.
+ */
+function render_block_core_playlist( $attributes, $content ) {
+	if ( empty( $attributes['tracks'] ) ) {
+		return '';
+	}
+
+	$current_id = $attributes['currentTrack'];
+
+	/**
+	 * Return early if no valid track ID is found.
+	 * This can happen if the user deleted all tracks but kept an empty inner block, such as the media upload placeholder.
+	 */
+	if ( empty( $current_id ) ) {
+		return '';
+	}
+
+	wp_enqueue_script_module( '@wordpress/block-library/playlist/view' );
+
+	$attachment_meta = wp_get_attachment_metadata( $current_id );
+	$current_title   = get_the_title( $current_id ) ? get_the_title( $current_id ) : '';
+	$current_artist  = isset( $attachment_meta['artist'] ) ? $attachment_meta['artist'] : '';
+	$current_album   = isset( $attachment_meta['album'] ) ? $attachment_meta['album'] : '';
+	$current_image   = isset( $attachment_meta['poster'] ) ? $attachment_meta['poster'] : '';
+	$current_url     = wp_get_attachment_url( $current_id );
+	$show_images     = isset( $attributes['showImages'] ) ? $attributes['showImages'] : false;
+	$aria_label      = $current_title;
+
+	if ( $current_title && $current_artist && $current_album ) {
+		$aria_label = sprintf(
+			/* translators: %1$s: track title, %2$s artist name, %3$s: album name. */
+			_x( '%1$s by %2$s from the album %3$s', 'track title, artist name, album name' ),
+			$current_title,
+			$current_artist,
+			$current_album
+		);
+	}
+
+	wp_interactivity_state(
+		'core/playlist',
+		array(
+			'currentID'     => $current_id,
+			'currentURL'    => $current_url,
+			'currentTitle'  => $current_title,
+			'currentAlbum'  => $current_album,
+			'currentArtist' => $current_artist,
+			'currentImage'  => $current_image,
+			'ariaLabel'     => $aria_label,
+		)
+	);
+
+	// Add the markup for the current track
+	$html = '<div><div class="wp-block-playlist__current-item">';
+	if ( $show_images && $current_image ) {
+		$html .=
+		'<img
+			class="wp-block-playlist__item-image"
+			src={ ' . esc_url( $current_image ) . '}
+			alt=""
```

Comment:
I consider it decorative. It would be a very generic alt attribute otherwise, which would be the same for every image, that would not bring much value, such as "Track poster image" or similar.